configcat/docs
# Add a redirect rule from /csharp to /dotnet

### Description

### Short description about the applied changes.

1. **Configured a client side redirect rule that redirects users**

- from: https://configcat.com/docs/sdk-reference/csharp/
- to: https://configcat.com/docs/sdk-reference/dotnet/

using this plugin: <https://docusaurus.io/docs/api/plugins/@docusaurus/plugin-client-redirects#redirects>

2. **Removed `/sdk-reference/csharp.md`**

### How to test this change locally

1. Navigate to the `website` directory in your terminal.

2. Run the following command to build:

```
npm run build
```

3. Run the following command to serve the build from the previous step locally:

```
npm run serve
```

3. Then in your address bar: 

- Go to `/docs/sdk-reference/csharp/` 

- You should be auto redirected to `/docs/sdk-reference/dotnet/` 

### Requirement checklist

- [ ] I have checked the SNYK/Dependabot reports and applied the suggested changes.
- [ ] (Optional) I have updated outdated packages.
